packages/client/modules/userDashboard/components/OrgAuthentication/OrgAuthenticationSignOnUrl.tsx

```diff
@@ -0,0 +1,130 @@
+import styled from '@emotion/styled'
+import {ContentCopy} from '@mui/icons-material'
+import React from 'react'
+import makeMaxWidthMediaQuery from '~/utils/makeMaxWidthMediaQuery'
+import CopyLink from '../../../../components/CopyLink'
+import DialogTitle from '../../../../components/DialogTitle'
+import BasicInput from '../../../../components/InputField/BasicInput'
+import RowActions from '../../../../components/Row/RowActions'
+import SecondaryButton from '../../../../components/SecondaryButton'
+import {MenuPosition} from '../../../../hooks/useCoords'
+import useTooltip from '../../../../hooks/useTooltip'
+import {PALETTE} from '../../../../styles/paletteV3'
+
+const mobileBreakpoint = makeMaxWidthMediaQuery(420)
+
+const largeMobileBreakpoint = makeMaxWidthMediaQuery(540)
+
+const Section = styled('div')({
+  margin: '0 auto',
+  maxWidth: '608px',
+  padding: '0px 16px 12px 16px',
+  [mobileBreakpoint]: {
+    padding: '0 28px 12px 28px'
+  }
+})
+
+const InputSection = styled('div')({
+  display: 'flex',
+  padding: 0,
+  maxWidth: '560px',
+  marginLeft: '16px',
+  [mobileBreakpoint]: {
+    marginLeft: '16px'
+  }
+})
+
+const SubTitle = styled(DialogTitle)<{disabled: boolean}>(({disabled}) => ({
+  color: disabled ? PALETTE.SLATE_600 : PALETTE.SLATE_700,
+  fontSize: '16px',
+  padding: 0
+}))
+
+const Label = styled('div')<{disabled: boolean}>(({disabled}) => ({
+  color: disabled ? PALETTE.SLATE_600 : PALETTE.SLATE_700,
+  fontSize: '14px',
+  display: 'flex',
+  alignItems: 'center'
+}))
+
+const CopyButton = styled(SecondaryButton)({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  marginLeft: '16px',
+  marginRight: '8px',
+  color: PALETTE.SLATE_600,
+  padding: '0',
+  border: `1px solid ${PALETTE.SLATE_400}`,
+  borderRadius: '50px',
+  width: '40px',
+  height: '40px',
+  [mobileBreakpoint]: {
+    marginLeft: '0px'
+  },
+  [largeMobileBreakpoint]: {
+    marginLeft: '8px',
+    marginRight: '16px'
+  }
```

Comment:
+1 I think we can simplify
We can remove  `marginRight: '8px'`here at all. Add `marginRight: 16px` on parent container (as it is already has margin-left)

And if we need to change margin left on small devices we can keep only one breakpoint

```

  [mobileBreakpoint]: {
    marginLeft: '8px',
  }
  ```

Or just keep margins the same on all screens, in this case we don't need breakpoints at all